docs/themes.rst

```diff
@@ -81,10 +81,12 @@ articles        The list of articles, ordered descending by date.
 dates           The same list of articles, but ordered by date,
                 ascending.
 drafts          The list of draft articles
-tags            A list of (tag, articles) tuples, containing all
-                the tags.
+authors         A list of (author, articles) tuples, containing all
+                the authors.
```

Comment:
For consistency, maybe add `and corresponding articles (values)` here and below (with tags), and drop the `.`

Reply:
I guess I didn't notice any period usage consistency.  If there is a style guide, i can normalize the rest of the documentation.  Otherwise, I added the additional text in the table.